.jenkins/pytorch/win-build.sh

```diff
@@ -79,6 +79,17 @@ if "%REBUILD%"=="" (
   )
 )
 
+:: Install Clang-LLVM
+if "%REBUILD%"=="" (
+  if "%BUILD_ENVIRONMENT%"=="" (
+    curl.exe -k -O https://s3.amazonaws.com/ossci-windows/LLVM-6.0.0.7z
+  ) else (
+    aws s3 cp s3://ossci-windows/LLVM-6.0.1-rC331533.7z LLVM-6.0.1-rC331533.7z --quiet
+  )
+  7z x ./LLVM-6.0.1-rC331533.7z -o"C:\Program Files\LLVM"
```

Comment:
It would be great to add `-aoa` options to 7z here (similar to how we do it for magma above), so that existing files in `C:\Program Files\LLVM` are guaranteed to be rewritten. Otherwise local builds with this script might run into a prompt that asks whether to overwrite the existing files.